### DISCUSSION TITLE
Curated "Adwaita Black" does not show border for GTK3 apps. Not possible?

### DISCUSSION THREAD

**Discussion by HubKing** [GradienceTeam/Gradience#704] [Q&A] [answered]
I applied the curated "Adwaita Black" theme,

![image](https://user-images.githubusercontent.com/10900489/212987085-c9ca9970-7c18-4008-9c44-6b24ff6db48c.png)

and I checked both GTK4 and GTK3 in the apply dialogue, and I set adw-gtk3 (also tried adw-gtk3-dark) for the "Legacy apps" in Tweaks. But it seems the border is only drawn for GTK4 apps. Below is the new Gnome text editor, which probably is using GTK4, and the old Gedit, which is probably using GTK3. Only the former has the border. Is this a limitation of GTK3 and cannot be fixed?

![image](https://user-images.githubusercontent.com/10900489/212987851-1a48a8a7-24fe-4e18-b82f-b5a4eae1c3f1.png)

![image](https://user-images.githubusercontent.com/10900489/212987989-c49f728e-05a4-47d4-8cbf-3b04d3dbc15f.png)

**Comment by daudix** [comment, marked as answer]
Oh yes, I forgot about gtk3 apps 😅
